businesscentral-app-secret.yaml

```diff
@@ -8,12 +8,6 @@ metadata:
 labels:
   template: businesscentral-app-secret
 items:
-- kind: ServiceAccount
-  apiVersion: v1
-  metadata:
-    name: businesscentral-service-account
-  secrets:
```

Comment:
Not sure if it can be removed, if it is remove users can't instantiate the full or templates that uses https easily.

Reply:
It has been moved to each template where the service account was being used